src/utiltime.cpp

```diff
@@ -81,3 +82,18 @@ std::string DateTimeStrFormat(const char* pszFormat, int64_t nTime)
     ss << boost::posix_time::from_time_t(nTime);
     return ss.str();
 }
+
+std::string DurationToDHMS(int64_t nDurationTime)
+{
+    int seconds = nDurationTime % 60;
+    nDurationTime /= 60;
+    int minutes = nDurationTime % 60;
+    nDurationTime /= 60;
+    int hours = nDurationTime % 24;
+    int days = nDurationTime / 24;
+    if((hours == 0)&&(days == 0))
```

Comment:
Nit: Would be nice to have some extra whitespace around the `&&`.

Might also consider removing inner parenthesis as similar conditions in this project don't seem to use them (eg: [addresstablemodel.cpp](https://github.com/dashpay/dash/blob/master/src/qt/addresstablemodel.cpp#L155)).